web/auth/jwt.go

```diff
@@ -36,10 +36,17 @@ type TokenManager struct {
 }
 
 // NewTokenManager starts a new token manager. Will create a list with all tokens that need update.
-func NewTokenManager(db database.Database) (*TokenManager, error) {
+func NewTokenManager(db database.Database, dev *bool) (*TokenManager, error) {
```

Comment:
`dev *bool` should be `dev bool`. This way you can avoid passing `nil` elsewhere:

```go
// web/users_test.go
// with dev *bool
tm, err := auth.NewTokenManager(db, nil)
// with dev bool
tm, err := auth.NewTokenManager(db, false)
```

In `main.go` you would instead send the value of `dev`, similar to how you do for `RegisterRouter`.

```go
tokenManager, err := auth.NewTokenManager(db, *dev)
```

However, if the comment below makes sense to use then the bool is not needed at all.

Reply:
See the alternative approach to set the secret below. That way you avoid changing the token manager signature.